AddQtIosApp.cmake

```diff
@@ -99,6 +100,7 @@ function(add_qt_ios_app TARGET)
         CATALOG_APPICON
         CATALOG_LAUNCHIMAGE
         DISTRIBUTION_METHOD
+        PHOTO_LIBRARY_USAGE_DES
```

Comment:
Can you use long variable name : `PHOTO_LIBRARY_USAGE_DESCRIPTION`.
I know it's long but when going back to it after month it's quite nice to understand description and not spending understanding what DES is. (Maybe Des is very common in english, but i never saw it as french person). Desc is more common to me.
What do you think?

Reply:
I agree let's go for `PHOTO_LIBRARY_USAGE_DESCRIPTION`